src/core/lib/interfaces/bdd.ts

```diff
@@ -14,21 +14,40 @@ import {
 
 export interface BddInterface extends TddLifecycleInterface {
   describe(name: string, factory: TddSuiteFactory): void;
+  xdescribe(name: string, factory: TddSuiteFactory): void;
   it(name: string, test: TestProperties['test']): void;
+  xit(name: string, test?: TestProperties['test']): void;
 }
 
 export { suite as describe };
 export { test as it };
 export { before, after, beforeEach, afterEach };
 
+export function xdescribe(name: string, factory: TddSuiteFactory) {
+  return suite(name, function(suiteRef) {
+    before(function() {
+      suiteRef.skip();
+    });
+    factory(suiteRef);
+  });
+}
+
+export function xit(name: string, _test?: TestProperties['test']) {
+  return test(name, function() {
+    this.skip();
```

Comment:
This should provide a skip reason to give the user some indication of why the suite was skipped ("xit", "not implemented", ...).